io.github.magnush.mhWaveEdit.yml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.magnush.mhWaveEdit
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: mhwaveedit
+rename-desktop-file: mhwaveedit.desktop
+
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=xdg-music
+  - --socket=pulseaudio
+  
+modules:
+- shared-modules/gtk2/gtk2.json 
+    
+- name: mhwaveedit
+  cleanup: 
+    - /cache
+    - /man
+    - /share/aclocal
+    - /share/devhelp
+    - /include
+    - /lib/pkgconfig
+    - /share/gtk-doc
+    - /share/man
+    - /lib/*.a
+    - '*.a'
+    - '*.la'
+
+  buildsystem: autotools
+
+  sources:
+  - type: git
+    url: https://github.com/magnush/mhwaveedit
+    commit: b0ae1edb86456f363e283bd740bbae01d58ae55c
+    tag: v1.4.24
+  - type: file
+    path: mhwaveedit128.png
+  - type: file
+    path: io.github.magnush.mhWaveEdit.metainfo.xml
```

Comment:
You also need to add something like this:
https://github.com/flathub/org.gimp.GIMP/blob/master/org.gimp.GIMP.json#L395
to build on aarch64

Reply:
is this good
```
  buildsystem: autotools

  sources:
  - type: git
    url: https://github.com/magnush/mhwaveedit
    commit: b0ae1edb86456f363e283bd740bbae01d58ae55c
    tag: v1.4.24

  - type: shell
    commands:
    - cp -p /usr/share/automake-*/config.{sub,guess} .

  - type: file
    path: mhwaveedit128.png
  - type: file
    path: io.github.magnush.mhWaveEdit.metainfo.xml
    
```